db/10_features_update.js

```diff
@@ -58,7 +58,8 @@ if(IMPOSM_ENABLED) {
 }
 
 Object.entries(projects).forEach(e => {
-	const [ id, project ] = e;
+	const [ id, name, project ] = e;
```

Comment:
This can't work, Object.entries returns a key -> value structure, so it's more a `[ name, project ]` that is returned there.

Reply:
```suggestion
	const [ name, project ] = e;
	const id = project.id;
	const slug = name.split("_").pop();
```
should work